src/components/MeritComment.svelte

```diff
@@ -0,0 +1,94 @@
+<script lang="ts">
+	import * as Card from '$lib/components/ui/card/index.js';
+	import { type MeritRequest } from '@/event_helpers/merits';
+	import { ndk } from '@/ndk';
+	import { NDKEvent, type NDKKind } from '@nostr-dev-kit/ndk';
+	import { Avatar, Name } from '@nostr-dev-kit/ndk-svelte-components';
+	import { onDestroy } from 'svelte';
+
+	import { Separator } from '$lib/components/ui/separator/index.js';
+	import { unixToRelativeTime } from '@/helpers';
+	import { Button } from '$lib/components/ui/button/index.js';
+	import { Label } from '$lib/components/ui/label/index.js';
+	import { Textarea } from '$lib/components/ui/textarea/index.js';
+	import { currentUser } from '@/stores/session';
+	import CornerDownLeft from 'lucide-svelte/icons/corner-down-left';
+	import type NDKSvelte from '@nostr-dev-kit/ndk-svelte';
+	export let merit: MeritRequest;
+
+	let comments = $ndk.storeSubscribe({ kinds: [1 as NDKKind], '#e': [merit.ID] });
+
+	onDestroy(() => {
+		comments.unsubscribe();
+	});
+
+	let content: string;
+
+	function publish(ndk: NDKSvelte) {
+		if (!ndk.signer) {
+			throw new Error('no ndk signer found');
+		}
+		let e = new NDKEvent(ndk);
+		let author = $currentUser;
+		if (!author) {
+			throw new Error('no current user');
+		}
+		e.author = author;
+		e.kind = 1;
```

Comment:
I did not see the need for a new kind to be used for merit discuss in NIP, so I used the generic kind 1 here.